tools/internal_ci/linux/grpc_tsan_on_foundry.sh

```diff
@@ -0,0 +1,87 @@
+Skip to content
+This repository
+Search
+Pull requests
+Issues
+Marketplace
+Explore
+ @adelez
+ Sign out
+ Unwatch 889
+  Star 13,284  Fork 2,893 grpc/grpc
+ Code  Issues 865  Pull requests 256  Projects 3  Wiki  Insights
+Branch: master Find file Copy pathgrpc/tools/internal_ci/linux/grpc_bazel_on_foundry_opt.sh
+54a7040  on Dec 14, 2017
+@apolcyn apolcyn Move more special cases in bazel build to the .bzl files
+2 contributors @apolcyn @adelez
+RawBlameHistory    
+57 lines (50 sloc)  2.14 KB
+#!/usr/bin/env bash
+# Copyright 2017 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+# A temporary solution to give Kokoro credentials. 
```

Comment:
Can this be changed to a TODO if we know what the future solution might be?

Reply:
Done